core/packages/test/scripts/build-binary.sh

```diff
@@ -32,7 +32,11 @@ rebuild_cumulus(){
 
 build_cumulus_from_source(){
     pushd $root_dir/cumulus
-    cargo build --release --bin polkadot-parachain
+    if [ "$active_spec" == "minimal" ]; then
```

Comment:
Double-square brackets are preferable. See https://stackoverflow.com/a/669486.
```suggestion
    if [[ "$active_spec" == "minimal" ]]; then
```